File: public/dsa/arrays/kth_largest_element/naive/kth_largest_element.py
from typing import List


class Solution:
    def getKthLargestElement(self, arr: List[int], k: int) -> int:
        arr.sort()
        i = len(arr) - k
        return arr[i]


# tests-start
arr = [4, 3, 6, 4, 1]
k = 3
sol = Solution()
sol.getKthLargestElement(arr, k)
# tests-end

'''output
4
'''

File: public/dsa/arrays/kth_largest_element/naive/kth_largest_element.cpp
#include <bits/stdc++.h>
using namespace std;

int getKthLargestElement(vector<int>& list, int k) {
    sort(list.begin(), list.end());
    int i = list.size() - k;
    return list[i];
}

// tests-start
int main() {
    vector<int> list = {4, 3, 6, 4, 1};
    int k = 3;
    int result = getKthLargestElement(list, k);
    cout << result << endl;
    return 0;
}
// tests-end

/*output
4
*/